### PR TITLE
run: copy .install files to the build directory

### PR DESCRIPTION
**Pull request by house-of-vanity**
Now this Action respects .install script which may bundled with PKGBUILD file.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -19,6 +22,8 @@ sed -i "s|MAKEFLAGS=.*|MAKEFLAGS=-j$(nproc)|" /etc/makepkg.conf
 useradd -m user
 cd /home/user
 
+# Copy PKGBUILD and *.install scripts
+cp $PKGBUILD_DIR/*install ./
```
Comment (FFY00) [inline run.sh]: Quote `$PKGBUILD_DIR` to prevent word splitting.

**Review by FFY00 — CHANGES_REQUESTED** [review]

**Comment by FFY00** [comment]
About the 2nd patch, why do you need it? I would expect action to replace files. Otherwise, if you already have a built package or logs there, you would get the old one.

File: run.sh [hunk 2]
```diff
@@ -19,6 +22,8 @@ sed -i "s|MAKEFLAGS=.*|MAKEFLAGS=-j$(nproc)|" /etc/makepkg.conf
 useradd -m user
 cd /home/user
 
+# Copy PKGBUILD and *.install scripts
+cp "$PKGBUILD_DIR/*install" ./
```
Comment (FFY00) [inline run.sh]: This doesn't work. `*` will not get expanded. Please change to `"PKGBUILD_DIR"/*.install`.

**Review by FFY00 — CHANGES_REQUESTED** [review]

**Comment by house-of-vanity** [comment]
> About the 2nd patch, why do you need it? I would expect action to replace files. Otherwise, if you already have a built package or logs there, you would get the old one.

If you set `INPUT_OUTDIR=./` so final step failed because of copying files into themselves. But as you noticed it's more clear when files are replaced.
Finally I decided to use default of INPUT_OUTDIR. Let me roll it back.

File: run.sh (hunk 2 above)
Comment (house-of-vanity) [reply run.sh]: Oh, of course. Sorry. 
Changed.

**Review by FFY00 — APPROVED** [review]

**Comment by FFY00** [comment]
> > About the 2nd patch, why do you need it? I would expect action to replace files. Otherwise, if you already have a built package or logs there, you would get the old one.
> 
> If you set `INPUT_OUTDIR=./` so final step failed because of copying files into themselves. But as you noticed it's more clear when files are replaced.
> Finally I decided to use default of INPUT_OUTDIR. Let me roll it back.

Hum, I understand. Not sure if we want to support this use case. I should definitely add a check though!

**Pull request merged**